function failing_test_function() {
	return 1
}